sites/geohub/src/routes/+layout.server.ts

```diff
@@ -4,8 +4,22 @@ import { DefaultUserConfig, type UserConfig } from '$lib/config/DefaultUserConfi
 
 export const load: LayoutServerLoad = async (event) => {
   const session = await event.locals.getSession()
-
-  const config: UserConfig = DefaultUserConfig
+  let config: UserConfig
+  try {
+    const response = await event.fetch('/api/settings', {
+      method: 'GET',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    })
+    if (response.ok) {
+      config = await response.json()
+    } else {
+      config = DefaultUserConfig
+    }
+  } catch (err) {
+    config = DefaultUserConfig
+  }
```

Comment:
Can you return DefaultUserConfig from `/api/settings` endpoints?

logic should be like below.
```js
if (!session) {
return DefaultUserConfig
} else {
return Object.assign(DefaultUserConfig, userConfig object)
}
```